grammar/Graql.g4

```diff
@@ -39,7 +39,7 @@ query_undefine      :   UNDEFINE    statement_type+ ;
 query_insert        :   MATCH       pattern+    INSERT  statement_instance+
                     |                           INSERT  statement_instance+  ;
 
```

Comment:
You should remove the new lines between insert and delete, and also make sure `;` are aligned.